playground/backend/internal/validators/python_validators.go

```diff
@@ -15,7 +15,7 @@
 
 package validators
 
-// GetPythonValidators return validators methods that should be applied to Go code
+// GetPythonValidators return validators methods that should be applied to Go javaCode
```

Comment:
`Go javaCode`? For  `GetPythonValidators`?

Reply:
Done.